<!DOCTYPE html>
<html lang="en">

  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

    <meta name="author" content="Rhys Newell">
    <meta name="description" content="Blogs, tools, and publications">
    <meta name="keywords" content="blog,developer,personal">

    <meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="Projects"/>
<meta name="twitter:description" content="Lorikeet Lorikeet is a variant calling, strain genome resolving, and microdiversity assessment algorithm for metagenomics. The variant calling algorithm Lorikeet uses is a full re-implementation of the GATK HaplotypeCaller algorithm in Rust. It is both faster and easier to use with almost identical results. Lorikeet can make use of both long and short reads simultaneously, handling both the mapping and parallelization of the analysis of multiple genomes with minimal effort required of the user."/>

    <meta property="og:title" content="Projects" />
<meta property="og:description" content="Lorikeet Lorikeet is a variant calling, strain genome resolving, and microdiversity assessment algorithm for metagenomics. The variant calling algorithm Lorikeet uses is a full re-implementation of the GATK HaplotypeCaller algorithm in Rust. It is both faster and easier to use with almost identical results. Lorikeet can make use of both long and short reads simultaneously, handling both the mapping and parallelization of the analysis of multiple genomes with minimal effort required of the user." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://rhysnewell.github.io/projects/" /><meta property="article:section" content="" />





    
      <base href="https://rhysnewell.github.io/projects/">
    
    <title>
  Projects · rhysnewell
</title>

    
      <link rel="canonical" href="https://rhysnewell.github.io/projects/">
    

    <link href="https://fonts.googleapis.com/css?family=Lato:400,700%7CMerriweather:300,700%7CSource+Code+Pro:400,700" rel="stylesheet">
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.11.2/css/all.css" integrity="sha384-KA6wR/X5RY4zFAHpv/CnoG2UW1uogYfdnP67Uv7eULvTveboZJg0qUpmJZb5VqzN" crossorigin="anonymous" />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css" integrity="sha256-l85OmPOjvil/SOvVt3HnSSjzF1TUMyT9eV0c2BzEGzU=" crossorigin="anonymous" />

    
      
      
      <link rel="stylesheet" href="https://rhysnewell.github.io/css/coder.min.624134b411585efffadab6a91e7d0383f0d4e22ad49de3690eccbc96f528e670.css" integrity="sha256-YkE0tBFYXv/62rapHn0Dg/DU4irUneNpDsy8lvUo5nA=" crossorigin="anonymous" media="screen" />
    

    

    

    

    

    

    <link rel="icon" type="image/png" href="https://rhysnewell.github.io/images/favicon-32x32.png" sizes="32x32">
    <link rel="icon" type="image/png" href="https://rhysnewell.github.io/images/favicon-16x16.png" sizes="16x16">

    <meta name="generator" content="Hugo 0.89.4" />
  </head>

  
  
  <body class="colorscheme-light">
    <main class="wrapper">
      <nav class="navigation">
  <section class="container">
    <a class="navigation-title" href="https://rhysnewell.github.io/">
      rhysnewell
    </a>
    
    <input type="checkbox" id="menu-toggle" />
    <label class="menu-button float-right" for="menu-toggle"><i class="fas fa-bars"></i></label>
    <ul class="navigation-list">
      
        
          <li class="navigation-item">
            <a class="navigation-link" href="https://rhysnewell.github.io/about/">About</a>
          </li>
        
          <li class="navigation-item">
            <a class="navigation-link" href="https://rhysnewell.github.io/posts/">Blog</a>
          </li>
        
          <li class="navigation-item">
            <a class="navigation-link" href="https://rhysnewell.github.io/projects/">Projects</a>
          </li>
        
          <li class="navigation-item">
            <a class="navigation-link" href="https://rhysnewell.github.io/publications/">Publications</a>
          </li>
        
          <li class="navigation-item">
            <a class="navigation-link" href="https://rhysnewell.github.io/conferences/">Talks &amp; Awards</a>
          </li>
        
          <li class="navigation-item">
            <a class="navigation-link" href="https://rhysnewell.github.io/contact/">Contact me</a>
          </li>
        
      
      
    </ul>
    
  </section>
</nav>


      <div class="content">
        
  <section class="container page">
  <article>
    <header>
      <h1>Projects</h1>
    </header>

    <h2 id="lorikeethttpsrhysnewellgithubiolorikeet"><a href="https://rhysnewell.github.io/Lorikeet/">Lorikeet</a></h2>
<p><img src="https://rhysnewell.github.io/images/lorikeet_logo.png" alt=""></p>
<p>Lorikeet is a variant calling, strain genome resolving, and microdiversity assessment algorithm for metagenomics.
The variant calling algorithm Lorikeet uses is a full re-implementation of the GATK HaplotypeCaller algorithm
in Rust. It is both faster and easier to use with almost identical results. Lorikeet can make use of both long and short
reads simultaneously, handling both the mapping and parallelization of the analysis of multiple genomes with minimal effort
required of the user.</p>
<h2 id="rosellahttpsrhysnewellgithubiorosella"><a href="https://rhysnewell.github.io/rosella/">Rosella</a></h2>
<p><img src="https://rhysnewell.github.io/images/rosella.png" alt=""></p>
<p>Rosella is a metagenomic binning algorithm written in Rust and Python. It makes use of UMAP and HDBSCAN to generate
metagenomic assembled genomes from metagenome assemblies. It can both map and calculate coverage information for assemblies
and is capable of handling and long and short reads simultaneously.</p>
<h2 id="aviaryhttpsgithubcomrhysnewellaviary"><a href="https://github.com/rhysnewell/aviary">Aviary</a></h2>
<p>Aviary is a modular and easy to use to complete metagenomic workflow written in python wrapping around snakemake. Aviary
is capable of handling hybrid assembly of long and short reads from various sequencing platforms. Additionally, Aviary performs
robust binning using a large suite of metagenomic binners and ensemble binning. Finalized bins are also assessed for quality using
CheckM, and assigned taxonomic ranks using GTDB-tk. Various other statistics are also reports like recovered diversity at various steps
using <a href="https://github.com/wwood/singlem">SingleM</a> and coverage statistics using <a href="https://github.com/coverm">CoverM</a></p>
<h2 id="chip-rhttpsgithubcomrhysnewellchip-r"><a href="https://github.com/rhysnewell/chip-r">ChIP-R</a></h2>
<p>ChIP-R is a reproducibility assessment algorithm for ChIP-seq and ATAC-seq data. It allows the testing of multiple replicates at a time
allowing users to generate finalized sets of high confidence ChIP-seq and ATAC-seq peaks with ease. The publication can be found
<a href="https://www.sciencedirect.com/science/article/abs/pii/S0888754321001531">here</a></p>
<h2 id="contributions">Contributions</h2>
<p><a href="https://github.com/wwood/kingfisher-download">Kingfisher</a>
<a href="https://github.com/wwood/coverm">CoverM</a>
<a href="https://github.com/wwood/galah">Galah</a> logo</p>
<h2 id="side-projects">Side projects</h2>
<h3 id="nymph">Nymph</h3>
<p>Non-negative matrix factorization in rust</p>

  </article>
</section>


      </div>

      <footer class="footer">
  <section class="container">
    
      <p>Ikigai.</p>
    
    
      
        © 2019 - 2021
      
       Rhys Newell 
    
    
       · 
      Powered by <a href="https://gohugo.io/">Hugo</a> & <a href="https://github.com/luizdepra/hugo-coder/">Coder</a>.
    
    
       · 
      [<a href="https://github.com/luizdepra/hugo-coder/tree/"></a>]
    
  </section>
</footer>

    </main>

    

  </body>

</html>
